test/test_qml/test_message_model.cpp

```diff
@@ -0,0 +1,20 @@
+#include "test_messagel_mode.h"
+
+#include "message.h"
+
+TestCase::TestCase()
+{
+    QList<Message> testMessageList;
+    testMessageList.append(Message("Ivan",    "3621897462", "2013-02-05T19:23:12+00:00", "Andryukha, we have a corpse."));
+    testMessageList.append(Message("Nikolay", "2897430847", "2012-03-25T17:26:34+00:00", "Possibly a crime!"));
+    testMessageList.append(Message("Michail", "5465484984", "2011-04-15T13:25:45+00:00", "On the horses!"));
+
+    m_messageModel.addMessage (testMessageList);
+}
+
+void TestCase::qmlEngineAvailable(QQmlEngine *engine)
```

Comment:
```suggestion
void TestMessageModel::qmlEngineAvailable(QQmlEngine *engine)
```
предлагают переименовать класс пока под конкретный тест